src/main/java/duke/DukeApp.java

```diff
@@ -0,0 +1,137 @@
+
+        textUi = new TextUi();
+        storage = new StorageFile(System.getProperty("user.home") + "/Desktop");
+        taskList = new TaskList(storage.load());
+
+        scrollPane = new ScrollPane();
+        dialogContainer = new VBox();
+        scrollPane.setContent(dialogContainer);
+
+        userInput = new TextField();
+        Button sendButton = new Button("Send");
+
+        AnchorPane mainLayout = new AnchorPane();
+        mainLayout.getChildren().addAll(scrollPane, userInput, sendButton);
+
+        Scene scene = new Scene(mainLayout);
+
+        stage.setTitle("Duke");
+        stage.setResizable(false);
+        stage.setMinHeight(600.0);
+        stage.setMinWidth(400.0);
+
+        mainLayout.setPrefSize(400.0, 600.0);
+
+        scrollPane.setPrefSize(385, 535);
+        scrollPane.setHbarPolicy(ScrollPane.ScrollBarPolicy.NEVER);
+        scrollPane.setVbarPolicy(ScrollPane.ScrollBarPolicy.ALWAYS);
+
+        scrollPane.setVvalue(1.0);
+        scrollPane.setFitToWidth(true);
+
+        dialogContainer.setPrefHeight(Region.USE_COMPUTED_SIZE);
+
+        userInput.setPrefWidth(325.0);
+
+        sendButton.setPrefWidth(55.0);
+
+        AnchorPane.setTopAnchor(scrollPane, 1.0);
+
+        AnchorPane.setBottomAnchor(sendButton, 1.0);
+        AnchorPane.setRightAnchor(sendButton, 1.0);
+
+        AnchorPane.setLeftAnchor(userInput , 1.0);
+        AnchorPane.setBottomAnchor(userInput, 1.0);
+
+        sendButton.setOnMouseClicked((event) -> {
+            handleUserInput();
+        });
+
+        userInput.setOnAction((event) -> {
+            handleUserInput();
+        });
+
+        //Scroll down to the end every time dialogContainer's height changes.
+        dialogContainer.heightProperty().addListener((observable) -> scrollPane.setVvalue(1.0));
+
+        stage.setScene(scene);
+        stage.show();
+    }
+
+    /**
+     * Creates two dialog boxes, one echoing user input and the other containing Duke's reply and then appends them to
+     * the dialog container. Clears the user input after processing.
+     */
+    private void handleUserInput() {
+        Label userText = new Label(userInput.getText());
+        Label dukeText = new Label(getResponse(userInput.getText()));
+
+        boolean isExit = false;
+        final String userCommand = userInput.getText();
+        final Command command = new Parser().parseCommand(userCommand);
+        CommandResult result = command.execute(taskList, textUi, storage);
+        Label dukeResult = new Label(result.getFeedbackToUser());
+
+        if (command.isExit()) {
+            System.exit(0);
+        }
+
+        dialogContainer.getChildren().addAll(
+                DialogBox.getUserDialog(userText, new ImageView(user)),
+                DialogBox.getDukeDialog(dukeText, new ImageView(duke)),
+                DialogBox.getDukeDialog(dukeResult, new ImageView(duke))
+        );
+
+        userInput.clear();
+    }
+
+    /**
+     * Respond to user input.
```

Comment:
I think you could try following the JavaDoc format for some of these functions?

```suggestion
     * Responds to user input.
     *
     * @param input User input to respond to.
     * @return String response to input
```